### ISSUE TITLE
can not build a custom profile because the app crashes

### ISSUE THREAD

**Issue by DrugoBot** [Dijji/FileMeta#52]
Hi, I moved from Win 10 1607 LTSB to Win 10 1809 LTSC. If I try to drag a property to build a custom profile, File Meta Association Manager closes. The dragging of any property, only 2 millimeters triggers the closure of the program.
By now I'm used to using your application and I would not abandon it. You have a solution.

Cheers

**Comment by Dijji** [comment]
I have tested the same thing on my Windows 10 Professional 1809 system, with no problems. It also occurred to me to wonder whether it is to do with the .Net version installed, I also tried it on a Windows preview version with just the very latest .Net framework, but again, no problems.

Unfortunately, I don’t have any access to Enterprise versions of Windows, so I can’t test a direct equivalent.

The only other thing I can suggest is to have a look at the Application event log, to see if there’s anything there. You might also try uninstalling and reinstalling File Meta. Otherwise, it will be a question of installing developer tools on your system - all free, but not a trivial exercise.

Dijji
